### PR TITLE
fix: html / css fixes, minor typography

### PR DESCRIPTION
**Pull request by corpulent**

### CODE REVIEW

File: services/frontend/src/components/Modal/network/IPam.tsx [hunk 1]
```diff
@@ -120,11 +122,7 @@ const IPam = () => {
         renderField={(
           element: ReactElement,
           field: IFieldType
-        ): ReactElement => (
-          <Field style={{ paddingLeft: field.type === "records" ? 16 : 0 }}>
-            {element}
-          </Field>
-        )}
```
Comment (itssamuelrowe) [inline services/frontend/src/components/Modal/network/IPam.tsx:127]: Why did you remove this? It adds padding to records within records.

Comment (corpulent) [reply services/frontend/src/components/Modal/network/IPam.tsx:127]: Its not necessary with new the design I am slowly rolling out.

**Review by itssamuelrowe — APPROVED** [review]
